unix/README.md

```diff
@@ -26,7 +26,11 @@ following packages pre-installed:
     libjpeg8-dev
     libtiff5-dev
     libopenexr-dev
-    
+
+To enable the, as rendering, Simple DirectMedia Layer display also pre-install:
```

Comment:
Hm... If I already happen to know that I want Simple DirectMedia Layer, I may also already happen to know that I want libsdl-dev.

Maybe be a bit more clear about the fact that it is for the render preview. Also, do we still support X-Window as an alternative? If so, we might want to mention that, too. (Or do we need _both_ for the render preview to work?)